megatron/arguments.py

```diff
@@ -1114,6 +1114,8 @@ def _add_distributed_args(parser):
                        'a custom built image that support ring-exchange p2p.')
     group.add_argument('--local-rank', type=int, default=None,
                        help='local rank passed from distributed launcher.')
+    group.add_argument('--local_rank', type=int, default=None,
```

Comment:
Can't we simply add `'--local_rank'` alias on Line#1115?

Reply:
Yeah I just commit a PR to merge them.